### PR TITLE
[15.0][FIX] stock_picking_report_valued: Calculate price unit correctly con…

### PR DESCRIPTION
**Pull request by ao-landoo**
…sidering uom

We fixed the calculation error of price unit on the stock picking report valued.

Before: Directly got value of price unit without looking at the unit of measure on the sale order.

Now: It takes into account the conversion ratio between the unit of measure on the sale order and the unit of measure on the stock picking in order to get the real price unit.

### CODE REVIEW

**Review by bi-landoo — APPROVED** [review]
Functional Approval

**Comment by bi-landoo** [comment]
@pedrobaeza @ernestotejeda  ¿Could you review this fix please?

File: stock_picking_report_valued/models/stock_move_line.py [hunk 1]
```diff
@@ -49,11 +50,16 @@ def _compute_sale_order_line_fields(self):
         """
         for line in self:
             sale_line = line.sale_line
+            sale_line_uom = sale_line.product_uom
             price_unit = (
                 sale_line.price_subtotal / sale_line.product_uom_qty
                 if sale_line.product_uom_qty
                 else sale_line.price_reduce
             )
+            if sale_line_uom.uom_type == "bigger":
```
Comment (pedrobaeza) [inline stock_picking_report_valued/models/stock_move_line.py]: Please use `uom.uom` `convert` method for changing the quantity in the previous expression.

Comment (ao-landoo) [reply stock_picking_report_valued/models/stock_move_line.py]: I used `_compute_price` method as I found no `convert` method

File: stock_picking_report_valued/models/stock_move_line.py [hunk 2]
```diff
@@ -49,11 +50,13 @@ def _compute_sale_order_line_fields(self):
         """
         for line in self:
             sale_line = line.sale_line
+            sale_line_uom = sale_line.product_uom
             price_unit = (
                 sale_line.price_subtotal / sale_line.product_uom_qty
                 if sale_line.product_uom_qty
                 else sale_line.price_reduce
             )
+            price_unit = sale_line_uom._compute_price(price_unit, line.product_uom_id)
```
Comment (pedrobaeza) [inline stock_picking_report_valued/models/stock_move_line.py]:
```suggestion
            if sale_line_uom != line.product_uom_id:
                price_unit = sale_line_uom._compute_price(price_unit, line.product_uom_id)
```

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**